{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "UpdateInfo": null,
  "CTAs": [],
  "faultNumber": "40817451",
  "multipleFaultNumbers": null,
  "date": "2021-06-08T20:12:17",
  "region": "Poulton-Le-Fylde",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": null,
  "estimatedTimeOfRestorationMajority": "2021-06-08T20:12:17",
  "information": "",
  "actualTimeOfRestoration": "2021-06-08T20:12:17",
  "outageCentrePoint": {
    "lat": 53.855769,
    "lng": -2.999681
  },
  "outageLocations": null,
  "AffectedPostcodes": " FY6 7RB, FY6 7TR, FY6 7UA, FY6 7UD",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GM"
}